WalletWasabi/Services/SleepInhibitor.cs

```diff
@@ -91,26 +95,19 @@ private async Task PreventSleepAsync()
 	{
 		IPowerSavingInhibitorTask? task = _powerSavingTask;
 
-		if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux) || RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+		if (task is not null)
 		{
-			if (task is not null)
-			{
-				if (!task.Prolong(Timeout))
-				{
-					Logger.LogTrace("Failed to prolong the power saving task.");
-					task = null;
-				}
-			}
-
-			if (task is null)
+			if (!task.Prolong(Timeout))
 			{
-				Logger.LogTrace("Create new power saving prevention task.");
-				_powerSavingTask = await TaskFactory!().ConfigureAwait(false);
+				Logger.LogTrace("Failed to prolong the power saving task.");
+				task = null;
 			}
 		}
-		else
+
+		if (task is null)
```

Comment:
```suggestion
		else
```

Is there a reason why you changed the else here?